src/librustc_error_codes/error_codes/E0714.md

```diff
@@ -1,5 +1,19 @@
 A `#[marker]` trait contained an associated item.
 
+Erroneous code example:
+
+```ignore (features can't be used outside of nightly)
```

Comment:
are these tests run with stable? can we change that?

Reply:
I think they're not, we should test them like the others.